docker-compose.yml

```diff
@@ -0,0 +1,295 @@
+services:
+  # Jugalbandi Manager
+  api:
+    build:
```

Comment:
Fix images to opennyai

Reply:
opennyai images are giving error.
Same error across all the services
`flow-1      |     components["port"] = int(components["port"])
flow-1      |                          ^^^^^^^^^^^^^^^^^^^^^^^
flow-1      | ValueError: invalid literal for int() with base 10: 'None'`